py/client/pydeephaven/_console_service.py

```diff
@@ -49,10 +53,11 @@ def bind_table(self, table: Table, variable_name: str):
         if not table or not variable_name:
             raise DHError("invalid table and/or variable_name values.")
         try:
-            response = self._grpc_console_stub.BindTableToVariable(
+            response, call = self._grpc_console_stub.BindTableToVariable.with_call(
                 console_pb2.BindTableToVariableRequest(console_id=self.console_id,
                                                        table_id=table.ticket,
                                                        variable_name=variable_name),
                 metadata=self.session.grpc_metadata)
+            self.session.update_metadata(call.initial_metadata())
```

Comment:
Style comment: maybe no action for now, but.....

it's a little bit unfortunate that we have to remember to call `self.session.update_metadata(call.initial_metadata())` after every one of our grpc calls. It might be nice to wrap that in a helper method... or a method that forwards arguments to grpc and then does the metadata update for us.... I think that's what I did in C++

Anyway, the suggestion for now is to a least *think* about how we could make it nicer and a little more foolproof the next time we refactor this code.

Reply:
I did it; added wrappers.